src/ImageSharp/Formats/WebP/EntropyIx.cs

```diff
@@ -18,6 +18,8 @@ internal enum EntropyIx
 
         Palette = 4,
 
-        NumEntropyIx = 5
+        PaletteAndSpatial = 5,
```

Comment:
This was missing when compared to the reference version, something new they've added since this was started?

Reply:
good spot, yeah this is indeed new